ee/query-service/app/api/license.go

```diff
@@ -40,6 +40,7 @@ type billingDetails struct {
 		BillingPeriodEnd   int64   `json:"billingPeriodEnd"`
 		Details            details `json:"details"`
 		Discount           float64 `json:"discount"`
+		SubscriptionStatus string  `json:"subscriptionStatus"`
```

Comment:
Is this gonna break response struct in the frontend?

Reply:
No, It won't break. This is an addition to the existing structure.